composer.json

```diff
@@ -4,6 +4,8 @@
     "bin": [
         "bin/rector"
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "license": "MIT",
```

Comment:
Could you move these 2 in the end of the file? Just personal prefference I have in all `composer.json` not to clutter main composer json info :)